### PR TITLE
13360 sentry job error

### PR DESCRIPTION
**Pull request by kreek**
## Description of change
Updates `SentryJob` to not retry and use the standard logger to capture the original [`Raven::Event`](https://github.com/getsentry/raven-ruby/blob/v2.1.2/lib/raven/event.rb) details.

## Testing done
unit tests

## Testing planned
monitor staging logs

#### Unique to this PR
- [ ] Sets `SentryJob` retry to false
- [ ] Catches errors within the job and logs the original event details

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by patrickvinograd** [comment]
I know you mentioned this in a meeting but what is the behavior if there's a connectivity issue to Sentry? Is that addressed somewhere other than with the StandardError/retry logic?

**Comment by kreek** [comment]
@patrickvinograd from the looks of it any errors that happen when [calling transport.send_event](https://github.com/getsentry/raven-ruby/blob/v2.1.2/lib/raven/client.rb#L39-L46) are caught and then they're [failed send](https://github.com/getsentry/raven-ruby/blob/v2.1.2/lib/raven/client.rb#L97-L107) method just logs that an event didn't go through. So in the standard usage outside of a job maybe it's expected that some events don't get tracked? (seems weird for a monitoring tool)

**Comment by kreek** [comment]
Actually makes sense that's how it works it just shouldn't raise any errors at all 😄You can configure a callback that gets called in the failed_send method so that you can choose how to be alerted if the connection is down.

**Comment by patrickvinograd** [comment]
We could make that callback increment a statsd counter and monitor on that if we were worried about losing events. @wyattwalter do you think that's worthwhile or overkill?

File: app/workers/sentry_job.rb [hunk 1]
```diff
@@ -3,9 +3,18 @@
...
-  sidekiq_options queue: 'tasker'
+  sidekiq_options queue: 'tasker', retry: false
 
   def perform(event)
     Raven.send_event(event)
+  rescue StandardError => e
+    Rails.logger.error(
+      "Error performing SentryJob: #{e.message}",
+      original_event: {
+        culprit: event&.culprit,
```
Comment (wyattwalter) [inline app/workers/sentry_job.rb]: I had the "opportunity" to observe one of these today and noticed in the logs:

`{"host":"xxx","application":"vets-api-worker","timestamp":"2018-10-03T22:09:43.366651Z","level":"warn","level_index":3,"pid":3658,"thread":"96883340","name":"Sidekiq","message":"NoMethodError: undefined method `culprit' for #<Hash:0x000000000e233da0>"}`

Comment (kreek) [reply app/workers/sentry_job.rb]: @wyattwalter 😐if this PR isn't merged yet is that an error from within Raven? In this PR for safety's sake we could just log a simpler message.

Comment (wyattwalter) [reply app/workers/sentry_job.rb]: It's not merged, but I pushed this one out to a worker as a test to see if it would resolve the issue.

Comment (jholton) [reply app/workers/sentry_job.rb]: Been following this issue in passing, so I could be missing something, but...
Sentry (Raven) events are (should) always just be [Ruby hashes](https://docs.sentry.io/clients/ruby/config/#optional-settings) in Raven 2+, which means it'll never respond to `culprit`.
Should it look like this?
```
original_event: {
        culprit: event[:culprit],
        extra: event[:extra],
        backtrace: event[:backtrace]
}
```
BTW, it looks like [`culprit` is deprecated](https://docs.sentry.io/clientdev/attributes/)

**Comment by kreek** [comment]
We're also at sentry-raven 2.1.2 and the latest is 2.7.4 so maybe they fixed the issue?

**Pull request merged**